src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -29,10 +36,9 @@ def __init__(
         self.cocoa_sdk_crash_detector = sdk_crash_detector
         self.event_stripper = event_stripper
 
-    def detect_sdk_crash(self, event: Event) -> None:
-
+    def detect_sdk_crash(self, event: Event) -> Event:
```

Comment:
```suggestion
    def detect_sdk_crash(self, event: Event) -> Optional[Event]:
```

If you want to enforce correctness of type hints, add `src/sentry/utils/sdk_crashes` to mypy.ini and run `make backend-typing`.

Reply:
I will do that in an extra PR. I created an issue for that https://github.com/getsentry/sentry/issues/49938.